src/content/docs/infrastructure/host-integrations/host-integrations-list/dojo-io-integration.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Dojo.io integration
+tags:
+    - dojo.io integration
+    - dojo integration
+    - new relic integrations
+metaDescription: "The New Relic Dojo.io integration sends performance metrics and inventory data from your Dojo.io framework to the New Relic platform."
+---
+import dojodashboard from 'images/infrastructure_screenshot-crop_dojo-dashboard.png'
+
+With the help of New Relic's Dojo dashboard, you can track your application's core web vitals, user time on the site, and browser performance metrics like Initial page load, route change, throughput and page views with Javascript errors. Built with our infrastructure agent and our browser monitoring, the Dojo dashboard lets you view all your most important data in one place.
```

Comment:
It's not accurate to say it's built with our infrastructure agent, since that's optional. Maybe reword that last sentence to say "Using our browser monitor agent," or something similar. :)